libkbfs/interfaces.go

```diff
@@ -807,6 +823,22 @@ type DirtyBlockCache interface {
 	Shutdown() error
 }
 
+// DiskBlockCache caches blocks to the disk.
+type DiskBlockCache interface {
+	// Get gets a block from the disk cache.
+	Get(ctx context.Context, tlfID tlf.ID, blockID kbfsblock.ID) (
+		[]byte, kbfscrypto.BlockCryptKeyServerHalf, error)
+	// Put puts a block to the disk cache.
+	Put(ctx context.Context, tlfID tlf.ID, blockID kbfsblock.ID, buf []byte,
+		serverHalf kbfscrypto.BlockCryptKeyServerHalf) error
+	// Delete deletes some blocks from the disk cache.
+	Delete(ctx context.Context, tlfID tlf.ID, blockIDs []kbfsblock.ID) error
+	// Evict evicts some number of blocks from the disk cache.
+	Evict(ctx context.Context, tlfID tlf.ID, numBlocks int) error
```

Comment:
By what code should `Evict` be called?  Seems like it should only happen internally within the disk block cache, but maybe I'm missing a piece of a larger plan here.